### PR TITLE
DBTP-340 Add buildspec file for creating conduit task image

### PR DESCRIPTION
**Pull request by gabelton**
Follow up to previous PR, adding buildspec file for creating conduit task image

Also fixing an unrelated [security vulnerability](https://github.com/uktrade/copilot-tools/security/dependabot/5)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -31,6 +31,7 @@ schema = "0.7.5"
 cfn-flip = "1.3.0"
 moto = {extras = ["all"], version = "^4.1.12"}
 aiohttp = "^3.8.4"
+certifi = "^2023.07.22"
```
Comment (WillGibson) [inline pyproject.toml:34]: Why is this added? It doesn't seem relevant to the other changes to me.

Comment (WillGibson) [inline pyproject.toml:34]: Why is this added? It doesn't seem relevant to the other changes to me.

File: Dockerfile.conduit [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM public.ecr.aws/docker/library/postgres:latest
+ENTRYPOINT ["tail", "-f", "/dev/null"]
```
Comment (yusufsheiqh) [inline Dockerfile.conduit:2]: What is the purpose of this? This can be removed as we will be building an image and pushing to the ECR.

```suggestion

```

File: buildspec-ecr.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    commands:
+      - echo Build started on `date`
+      - echo Building the Docker image...
+      - docker build -t tunnel:latest --file Dockerfile.conduit .
+      - docker tag tunnel:latest public.ecr.aws/uktrade/tunnel:latest
+  post_build:
+    commands:
+      - echo Build completed on `date`
+      - echo Pushing the Docker image...
+      - docker push public.ecr.aws/uktrade/tunnel:latest
```
Comment (WillGibson) [inline buildspec-ecr.yml:18]: Can we add the line break at the end to stop GitHub moaning?

You can probably automate this on save in your IDE.

File: pyproject.toml (hunk 1 above)
Comment (WillGibson) [reply pyproject.toml:34]: Ah, I may have connected that if I was more intelligent, but as I'm not, maybe explicitly mentioning `certifi` in the description would have saved me starting this thread.

File: buildspec-ecr.yml
Comment (yusufsheiqh) [inline buildspec-ecr.yml:1]: I reckon we can rename this file to `buildspec.conduit.yml` or something similar.

**Review by yusufsheiqh — APPROVED** [review]
